recipes/mms-python-adapter/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "mms-python-adapter" %}
+{% set version = "1.2.5" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
```

Comment:
I think you can remove a lot of those jinja variables and stay just with name and version